cloud.profiles.d/edx.yml

```diff
@@ -0,0 +1,21 @@
+edx:
+  provider: mitx
+  size: m3.xlarge
```

Comment:
Can we get away with running on t2.large? Even if we have up to 6 of them it's still cheaper than 3 of the xlarge.